_posts/en/newsletters/2024-11-29-newsletter.md

```diff
@@ -0,0 +1,221 @@
+    conditions.  An alternative condition may also be satisfied to allow
+    spending the entire UTXO; for example, this could allow all parties
+    to agree to update some of the conditions.  This is flexible type of
+    [covenant][topic covenants] mechanism, similar to Towns's previous
+    proposal for `OP_TAP_LEAF_UPDATE_VERIFY` (TLUV, see [Newsletter
+    #166][news166 tluv]), but Towns has [written previously][towns
+    badcov] that he thinks _covenants_ is "not an accurate or useful
+    term".
+
+    Several examples for how these _flexible coin earmarks_ can be used
+    are provided, including improvements in the security and usability
+    of LN channels (including [LN-Symmetry][topic eltoo]-based
+    channels), an alternative to the [BIP345][] version of [vaults][topic
+    vaults], and a [payment pool][topic joinpools] design similar to
+    that contemplated for use with TLUV but that avoids the problems
+    that proposal had with [x-only public keys][topic x-only public
+    keys].
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [How does ColliderScript improve Bitcoin and what features does it enable?]({{bse}}124690)
+  Victor Kolobov lists potential uses for ColliderScript (see [Newsletter
+  #330][news330 cs] and [Podcast #330][pod330 cs]) including [covenants][topic
+  covenants], [vaults][topic vaults], emulation of [CSFS][topic
+  op_checksigfromstack], and validity rollups (see [Newsletter #222][news222
+  validity rollups]) while noting the high computational costs of such transactions.
+
+- [Why do standardness rules limit transaction weight?]({{bse}}124636)
+  Murch provides arguments for and against Bitcoin Core's standardness weight
+  limits and outlines how economic demand for larger transactions could erode
+  the effectiveness of the [policy][policy series].
+
+- [Is the scriptSig spending an PayToAnchor output expected to always be empty?]({{bse}}124615)
+  Pieter Wuille points out that because of how [pay-to-anchor (P2A)][topic ephemeral
+  anchors] outputs are [constructed][news326 p2a], they must adhere to segwit spending
+  conditions, including an empty scriptSig.
+
+- [What happens to the unused P2A outputs?]({{bse}}124617)
+  Instagibbs notes that today, unused P2A outputs with values at or above
+  Bitcoin Core's 240 satoshi [dust][topic uneconomical outputs] limit will
+  eventually be swept, removing them from the UTXO set. He goes on to reference
```

Comment:
I don't like this wording because it suggests P2A outputs smaller than 240 sats will not be swept. How about something like:
```suggestion
  Instagibbs notes that unused P2A outputs will eventually be swept when the block
  inclusion feerate drops low enough to make a sweep worth it, removing them from
  the UTXO set. He goes on to reference
```